.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI for React App
+
+on:
+  push:
+    branches: [ test/testing-jest-features ]
```

Comment:
You can use glob patterns to define branches to run on. I suggest defining the the patterns over feature and fix branches rather than using this branch name since it will definitely be a short lived branch.